Tools/LambdaTestTool-v2/src/Amazon.Lambda.TestTool/Utilities/HttpRequestUtility.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace Amazon.Lambda.TestTool.Utilities;
+
+/// <summary>
+/// Utility class for handling HTTP requests in the context of API Gateway emulation.
+/// </summary>
+public static class HttpRequestUtility
+{
+    /// <summary>
+    /// Determines whether the specified content type represents binary content.
+    /// </summary>
+    /// <param name="contentType">The content type to check.</param>
+    /// <returns>True if the content type represents binary content; otherwise, false.</returns>
+    public static bool IsBinaryContent(string? contentType)
+    {
+        if (string.IsNullOrEmpty(contentType))
+            return false;
+
+        return contentType.StartsWith("image/", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
We have this list of content types we use for response in the ASP.NET Core bridge library. https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs#L59

Reply:
We should also think about how this can be configurable. Probably via environment variables that we can have users configure in Aspire.